dist/mapd3.css

```diff
@@ -96,15 +90,9 @@
     border-collapse: separate;
     border-spacing: 0 5px; }
   .mapd3 .tooltip-group .tooltip-item, .mapd3 .legend-group .tooltip-item {
-    display: -webkit-box;
```

Comment:
@uyanga-gb looks like you may have forgotten to install autoprefixer which is why these were removed. Try doing a `yarn install` and then rebuilding the `dist`

Reply:
@uyanga-gb actually this is happening on a PR I have open on mapd3 as well, looking into why...

Reply:
@clhenrick has a solution here (for autoprefixer to also run on dev build) https://github.com/mapd/mapd3/pull/56

Reply:
yup! Just merged it into `master` so when you merge with master just make sure to rebuild `dist`